report-viewer/src/views/OverviewView.vue

```diff
@@ -194,6 +196,14 @@ function getMetricFromNumber(metric: number) {
   }
 }
 
+function getIndexFromMetric(metric: MetricType) {
+  if (metric == MetricType.AVERAGE) {
+    return 0
+  } else {
+    return 1
+  }
+}
```

Comment:
Is it possible in TS to define that more extensible ? Seems like some hard-coding :)